### ISSUE TITLE
'npm run coverage' does not work

### ISSUE THREAD

**Issue by jrobbins** [GoogleChrome/chromium-dashboard#1208]
**Describe the bug**
We should be able to run 'npm run coverage' to get a unit test coverage report.

**To Reproduce**
1. 'npm run coverage'

It gives an error 'No module named coverage'.  So, we probably need to add something to requirements.dev.txt.

**Comment by whoanuragverma** [comment]
I was able to run coverage, however none of the tests are passing. Please look into this.

**Comment by hemcharan22** [comment]
"test:ci": "cross-env CI=true react-scripts test --forceExit --detectOpenHandles",

Explanation: The problem was coming from Jest not being able to close all processes. The above is a quick workaround. Ideally you should track the process that's stopping Jest from exiting.

found this in a stackoverflow post

**Comment by shivamag00** [comment]
@jrobbins I have raised the PR - #1212 which should fix this issue. I have explained all the errors and failures that were raised while running coverage [here](https://github.com/GoogleChrome/chromium-dashboard/pull/1212#issue-594635706)

**Comment by codereverma** [comment]
@shivamag00 If someone has already raised a PR for a specfic issue then if something's wrong or there is bug in their PR, you **review it and comment**, like "hey, i think this creates a new bug" or "hey you can include this in your PR this will make it better" not **MAKE A NEW PR** with their changes and some other stuff that you did.

Open Source is for <ins>**collaboration**</ins> not <ins>**competition**</ins>. 

Last year also during hacktoberfest I saw people randomly creating PR by just taking someone else's. I was so pissed that I deleted my GitHub. And by the way all of those spammy PRs were for a free t-shirt whereas this time you are going to get ample amount of money. 

If you don't know how to contribute, please read [this](https://opensource.guide/).

**Comment by shivamag00** [comment]
@codereverma , I sincerely apologise if I offended you in any way. I think there is some confusion and would like to clear the same. @whoanuragverma  himself asked to look into the reason behind failing tests. Since I have just started contributing to this project and there were many changes that needed to be done to fix that issue, I was not sure if that was the correct and the most efficient way. Moreover, it was not possible to communicate all those changes to @whoanuragverma  through comments. But still, in the PR that I raised, I mentioned the contribution of @whoanuragverma and @jrobins for that issue. 

I hope this clears the confusion.
